packages/rn-tester/Podfile

```diff
@@ -64,7 +64,9 @@ pre_install do |installer|
   frameworks_pre_install(installer) if ENV['USE_FRAMEWORKS'] == '1'
   if ENV['USE_CODEGEN'] != '0'
     prefix_path = "../.."
-    codegen_pre_install(installer, {path:prefix_path})
+    codegen_path = "../../packages/react-native-codegen"
+    system("./#{codegen_path}/scripts/oss/build.sh") or raise "Could not build react-native-codegen package"
+    codegen_pre_install(installer, {path:prefix_path, codegen_path:codegen_path})
```

Comment:
I’m unsure why codegen is still invoked at all as part of `pod install`. Having it be part of the actual build pipeline means it will always run when necessary, right? So then it seems needlessly redundant to me to still invoke it here too.

Reply:
We won't need to generate these as a pre-install hook when they are part of the build pipeline, I agree. I'll send an update soon that moves everything to the FBReactNativeSpec podspec and out of the Podfile.